scripts/zones/Ruhotz_Silvermines/mobs/Guivre

```diff
@@ -0,0 +1,67 @@
+-----------------------------------
+-- Area: BCNM
+-- NPC:  Guivre
+-----------------------------------
+
+require("scripts/globals/titles");
+require("scripts/globals/status");
+	local path = 
+	{
+	-79, 0 , 382,
+	-99, 0 , 372,
+	-100,0,354,
+	-103,0,345,
+	-101,0,328,
+	-99,0,308,
+	-89,0,300,
+	-66,0,300,
+	-60,0,310,
+	-59.6,0,328,
+	-54,0,338,
+	-59.5,0,347,
+	-60,0,355,
+	-59,0,368,
+	-70.7,0,379, 
+	};
+
+function onMobSpawn(mob)
+	
+	onMobRoam(mob);
+
+	mob:speed(90);
+	
+end;
+
+function onPath(mob)
+	
+	pathfind.patrol(mob, path);
+	if (mob:getHP() < ((mob:getMaxHP() / 10) * 9.0) and BattleTime == 50) then
+	end
+	
+	
+end;
+
+function onMobRoam(mob)
+	-- move to start position if not moving
+	if (mob:isFollowingPath() == false) then
+		mob:pathThrough(pathfind.first(path));
+		
+	end
+	
+end;
+-----------------------------------
+-- onMobFight
+-----------------------------------
+function onMobFight(mob,target)
+		
+			
+		
+end;
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+function onMobDeath(mob,killer)	
```

Comment:
@Musashi-  this was changed, just FYI.

its now `function onMobDeath(mob, killer,ally)`for the function and `ally:doShit()` and `doShit(ally)` for most things. Killer will **only** effect person who did killing blow. `ally` runs for everybody like `killer` used to.

And please don't put in extra whitespace for no reason. Among other things I literately just got 300 or so scripts cleaned up the other day so they'd all conform to a single styling with identical headers and spacing, I really don't want to see the old stuff making a comeback. https://github.com/DarkstarProject/darkstar/blob/0307e011f10bb108b6bbc6bf81731ea9f3e7608d/scripts/zones/Kuftal_Tunnel/mobs/Guivre.lua